pyproject.toml

```diff
@@ -1,3 +1,73 @@
+[project]
+name = 'scylla-cluster-tests'
+version = '0.1'
+requires-python = ">=3.10, <3.14"
+dependencies = [
+    "aexpect==1.6.2",
+    "anyconfig==0.12.0",
+    "awscli==1.29.4",
+    "azure-identity==1.16.1",
+    "azure-mgmt-compute==33.1.0",
+    "azure-mgmt-network==28.1.0",
+    "azure-mgmt-resource==23.2.0",
+    "azure-mgmt-resourcegraph==8.0.0",
+    "azure-mgmt-subscription==3.1.1",
+    "azure-storage-blob==12.16.0",
+    "boto3-stubs[s3,ec2,dynamodb,pricing]==1.28.4",
+    "boto3==1.28.4",
+    "click==8.1.3",
+    "confluent-kafka==2.5.3",
+    "deepdiff==8.1.1",
+    "docker==7.1.0",
+    "elasticsearch==8.17.0",
+    "fabric==3.2.2",
+    "fastavro==1.9.7",
+    "gimme-aws-creds==2.8.2",
+    "google-api-python-client==2.93.0",
+    "google-cloud-compute==1.13.0",
+    "google-cloud-storage==2.10.0",
+    "hdrhistogram==0.9.2",
+    "invoke==2.2.0",
+    "jinja2==3.1.6",
+    "kubernetes==31.0.0",
+    "ldap3==2.9.1",
+    "moto[server]==5.1.1",
+    "mysql-connector-python==9.1.0",
+    "packaging>=24",
+    "paramiko==3.5.0",
+    "parameterized==0.8.1",
+    "pbr==6.1.0",
```

Comment:
When a python project really uses the `pbr` package then this package should be the first one in the list of a python project dependencies.

But, as I can see, its usage is not enabled because we would need something like following in this file:

```
[build-system]
requires = [
    "pbr>=6.1.0",
    "setuptools>=76.1.0"
]
build-backend = "pbr.build"
```

So, we should either remove the `pbr` dependency as unused or start really using it.

Reply:
Nice catch, I have no idea about why we wanted pbr here, so I will remove it